apollo-router/src/uplink/mod.rs

```diff
@@ -154,7 +153,7 @@ pub(crate) async fn fetch_supergraph(
         None => match http_request(GCP_URL, &request_body).await {
             Ok(response) => response,
             Err(e) => {
-                tracing::error!("could not get schema from GCP, trying AWS: {:?}", e);
+                tracing::debug!("could not get schema from GCP, trying AWS: {:?}", e);
```

Comment:
that's still a valuable signal. Could that log be displayed after multiple failures? And then, not displayed again until we get a successful download, for which we would have an info level message?